xdem/coreg.py

```diff
@@ -2155,134 +2180,233 @@ def warp_dem(
     return warped.reshape(dem.shape)
 
 
-hmodes_dict = {
-    "nuth_kaab": NuthKaab(),
-    "nuth_kaab_block": BlockwiseCoreg(coreg=NuthKaab(), subdivision=16),
-    "icp": ICP(),
-}
+def create_inlier_mask(
+    src_dem: RasterType,
+    ref_dem: RasterType,
+    shp_list: list[str | gu.Vector | None] | tuple[str | gu.Vector] | tuple[()] = (),
+    inout: list[int] | tuple[int] | tuple[()] = (),
+    filtering: bool = True,
+    dh_max: AnyNumber = None,
+    nmad_factor: AnyNumber = 5,
+    slope_lim: list[AnyNumber] | tuple[AnyNumber, AnyNumber] = (0.1, 40),
+) -> NDArrayf:
+    """
```

Comment:
Generally, the docstring of `dem_coregistration()` reads much better than this one at the same keys. Mind copying the `dem_coregistration()` versions here?

In the future, perhaps we should have a system like pandas where documentation can be shared among multiple functions. That makes it easier to be consistent!

Reply:
Done in 94c6a07

Yes, if you're motivated, I'm happy to see such an example !